jdaviz/configs/default/plugins/export_plot/export_plot.py

```diff
@@ -74,3 +105,153 @@ def vue_save_figure(self, filetype):
         the bqplot.Figure save methods.
         """
         self.save_figure(filetype=filetype)
+
+    def _save_movie(self, i_start, i_end, fps, filename, rm_temp_files):
+        # NOTE: All the stuff here has to be in the same thread but
+        #       separate from main app thread to work.
+        self.app.loading = True  # Grays out the app
+        viewer = self.viewer.selected_obj
+        slice_plg = self.app._jdaviz_helper.plugins["Slice"]._obj
+        orig_slice = slice_plg.slice
+        ts = float(slice_plg.play_interval) * 1e-3  # ms to s
+        temp_png_files = []
+
+        # TODO: Expose to users?
+        i_step = 1  # Need n_frames check if we allow tweaking
+
+        # No wrapping. Forward only.
+        if i_start < 0:
+            i_start = 0
+        if i_end > slice_plg.max_value:
+            i_end = slice_plg.max_value
+        if i_end <= i_start:
+            raise ValueError(f"No frames to write: i_start={i_start}, i_end={i_end}")

```

Comment:
Any case where an exception can be raised needs to return a functional app (not sure if there are others or not... perhaps the giant try except in `vue_export_movie`?)

```suggestion
            self.app.loading = False
            raise ValueError(f"No frames to write: i_start={i_start}, i_end={i_end}")

```

Reply:
Oh, oops. Good catch. I'll accept the suggestion first and see if a simple clean-up is possible later.